controls/radrichtextbox/import-export/Html/settings.md

```diff
@@ -0,0 +1,138 @@
+	* __Inline__: Inline properties of document elements using style attribute.	
+* __Title__: A property of type __string__ that allows you to set a Title to the generated HTML file. 
+
+#### __[C#] Example 1: Exclude properties__
+{{region c#-radrichtextbox-import-export-html-settings_0}}
+	 HtmlExportSettings htmlExportSettings = new HtmlExportSettings();
+    htmlExportSettings.PropertiesToIgnore["span"].Add("color");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("text-decoration");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("font-weight");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("font-style");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("font-family");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("font-size");
+    htmlExportSettings.PropertiesToIgnore["span"].Add("dir");
+
+    htmlExportSettings.PropertiesToIgnore["p"].Add("margin-top");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("margin-bottom");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("margin-left");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("margin-right");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("line-height");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("text-indent");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("text-align");
+    htmlExportSettings.PropertiesToIgnore["p"].Add("direction");
+
+    htmlExportSettings.PropertiesToIgnore["table"].Add("border-top");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("border-bottom");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("border-left");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("border-right");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("table-layout");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("margin-left");
+    htmlExportSettings.PropertiesToIgnore["table"].Add("border-spacing");
+
+    htmlExportSettings.PropertiesToIgnore["td"].Add("border-top");
+    htmlExportSettings.PropertiesToIgnore["td"].Add("border-bottom");
+    htmlExportSettings.PropertiesToIgnore["td"].Add("border-left");
+    htmlExportSettings.PropertiesToIgnore["td"].Add("border-right");
+    htmlExportSettings.PropertiesToIgnore["td"].Add("padding");
+    htmlExportSettings.PropertiesToIgnore["td"].Add("vertical-align");
+{{endregion}}
+
+>In order to achieve best compatibility of the generated HTML with __Telerik Reporting__, you should apply the following settings:  
+>* __DocumentExportLevel__: Fragment;  
+>* __StylesExportMode__: Inline;
+>* __StyleRepositoryExportMode__: DontExportStyles;
+>* __ExportFontStylesAsTags__: true.
+
+### ExportSettings Events
+* __CssClassExporting__: This event is fired on every attempt to export a CSS class.
+* __FloatingUIContainerExporting__: This event is fired on every attempt to export a __FloatingUIContainer__.
+* __ImageExporting__: This event is fired on every attempt to export an Image.
```

Comment:
This event is fired only when the respective option of ImageExportMode is applied.